pkg/slo-controller/noderesource/device_resource_calculator.go

```diff
@@ -136,6 +136,51 @@ func (r *NodeResourceReconciler) updateGPUNodeResource(node *corev1.Node, device
 	return err
 }
 
+func (r *NodeResourceReconciler) cleanupGPUNodeResource(node *corev1.Node) error {
+	deletedKeys := []corev1.ResourceName{
+		extension.ResourceGPU,
```

Comment:
Not only the resources of these GPUs, but also RDMA, FPGA, etc.

Reply:
RDMA, FPGA have been added.
RDMA, FPGA are not reported by koordlet now,right? other plugin?

Reply:
yes，koordlet does not report these devices but other plugins will report them。

Reply:
I've wondered before which resources need to be cleaned up, device crd has extended interfaces for other types of devices, code in https://github.com/koordinator-sh/koordinator/blob/main/apis/scheduling/v1alpha1/device_types.go#L26-L30 ,  but koord-manager only handles gpu type devices, code in https://github.com/koordinator-sh/koordinator/blob/main/pkg/slo-controller/noderesource/device_resource_calculator.go#L37-L59 
If there are plans to handle other devices via koordinater, they can also be left in, or I'll comment them out first, WDYT?

Reply:
Just deal with the GPU first.